packages/protocol/contracts/L1/tiers/TaikoA6TierProvider.sol

```diff
@@ -37,8 +37,8 @@ contract TaikoA6TierProvider is EssentialContract, ITierProvider {
         if (tierId == LibTiers.TIER_OPTIMISTIC) {
             return ITierProvider.Tier({
                 verifierName: "tier_optimistic",
-                validityBond: 1000 ether, // TKO
-                contestBond: 1000 ether, // TKO
+                validityBond: 250 ether, // TKO
+                contestBond: 500 ether, // TKO
```

Comment:
contestBond is higher now because if the original proof is invalid, then we don't want the proof to be contested; if the original proof is indeed valid, the contester wins 125 TKO, this means a 25% return for the 500 contestBond.